guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,435 @@
+4. The AI algorithm evaluates the image metadata with the pixels and determines if the metadata (i.e. left shoulder) is indeed what the pixel data is showing (classification for left shoulder from image data).
+5. After classifying the image the AI system sends a response back to the appropriate DICOM target location.
+6. After receiving the response the DICOM router can route the image accordingly, either to the rejected queue for further analysis by QA technicians, or if everything looks good to the radiology work queue.
+
+**Clinical Workflow (without DICOM Router)**
+
+1. Studies are initiated on a modality and the DICOM is sent directly to a generalized application that ingests raw DICOM files.
+2. The application will read the DICOM metadata and select a single image or slice from the series and send a significantly reduced dataset to the AI platform and algorithm specific for the QA study associated with the dataset. Note: in this example the reduction is in some aspect of the overall data set to simplify the AI processing complexity in proportion to the algorithm accuracy.   In some variations, the application will send several image metadata and pixel sets in parallel to an AI platform and parse the results collectively.
+3. The AI platform will evaluate the image metadata and pixels to validate the pair. In this example a classification solution is used and the results are sent back to the application.
+4. After classification results are sent back to the application it adds metadata to the DICOM images that can be used later to filter the correct series of images for radiology studies. These DICOMs are added to the PACs along with the original data and made available to QA technicians.
+5. Optionally and in addition to the previous action, the application can directly send corrected DICOM studies or augmented DICOMs to additional AI processing pipelines with different algorithms. 
+
+**Clinical Use Case:** Study Metadata QA
+
+**Computational Workload:**[Synchronous](#synchronous-computational-workload)
+
+**Clinical Workload:** 400 studies per hour, 1,200 series per hour, 1 slice per series, response time requirement of less than 500 milliseconds per slice.
+
+**Importance to Clinical AI Deployment:** critical, this type of inference is expected to be the most common across imaging, as it can drive a lot of other use cases downstream.
+
+**Figure:** Study Metadata QA without DICOM Router
+
+![MONAI_Metadata Quality Assurance](assets/MONAI_Metadata_Quality_Assurance.jpg)
+
+#### Image Quality Verification During Image Acquisition {#image-quality-verification-during-image-acquisition}
+
+Help the radiology technician improve their results.  During an image acquisition a QA score is provided to the radiology technician to help validate if the image quality is sufficient and if imaging needs to be re-taken.  Typically seconds or 10’s of seconds matter for efficient, high quality imaging.
```

Comment:
Hyeonhoon
On the 70th line of "Help the radiology technician improve their results.", it might be better to clarity when revised to "Help the radiology technician assure the image quality."

Reply:
Fixed